exporters/otlp/otlplogs/otlplogshttp/client.go

```diff
@@ -339,3 +339,7 @@ func (d *client) MarshalLog() interface{} {
 		Insecure: d.cfg.Insecure,
 	}
 }
+
+func init() {
+	otlplogs.Clients[otlpconfig.ExporterProtocolHttpProtobuf] = NewClient()
+}
```

Comment:
Same goes to here as well, the registration will be executed only once on the startup